README.md

```diff
@@ -86,8 +89,13 @@ with the Conda installation (sections [1a](#1a-anaconda) and [1b](#1b-starting-f
 
 1. Binary installation (Anaconda)
 
+The binary distribution of Sherpa 4.8.1 only supports Python 2.7.
+
 2. Source build (from a source tarball or the GitHub repository)
 
+A source build is - at present - the *only* way to use Sherpa 
```

Comment:
Let's remember to change this when we release, as at that point it won't probably be true any more ;)